components/layout/Layout.tsx

```diff
@@ -56,42 +57,42 @@ const MainArea = styled.div`
   @media (min-width: 980px) {
     margin-left: 25%;
```

Comment:
Could we also add a `margin-right: 30px` here so that the content doesn't extend towards the right? 
In a smaller screen, the content/banner would take up the whole screen which is fine. But in a bigger screen, I think it looks better with a little gap on the right like in the mock-ups.

Reply:
For sure! I was going back and forth; should be fixed [here](https://github.com/Dwolla/dev-portal/pull/40/commits/e452d15de54766a13e5e5e80cf5bf6e927ba2236)

Reply:
Sweet! Yeah, good call! Does look better when it's lined up with the button. 👍